tests/test_regression.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from cleanlab.regression import rank
+
+# To be used for all the tests
+labels = np.array([1, 2, 3, 4])
+predictions = np.array([1, 3, 4, 5])
+
+# test with deafault parameters
+def test_output_shape_type():
+    scores = rank.get_label_quality_scores(labels=labels, predictions=predictions)
+    assert labels.shape == scores.shape
+    assert isinstance(scores, np.ndarray)
+
+
+# test for acceptable datatypes
+@pytest.mark.parametrize("format", [pd.Series, pd.DataFrame, list])
+def test_type_error_for_input_types(format):
+    with pytest.raises(TypeError) as error:
+        _ = rank.get_label_quality_scores(labels=format(labels), predictions=format(predictions))
+
+
+# test for input shapes
+def test_assertion_error_for_input_shape():
+    with pytest.raises(AssertionError) as error:
+        _ = rank.get_label_quality_scores(labels=labels[:-1], predictions=predictions)
+        _ = rank.get_label_quality_scores(labels=labels, predictions=predictions[:-1])
```

Comment:
You know the dimensions of the `labels[:-1]` in the first line, so I'm sure you could make an assertion on the error message.

Also, can you confirm that we reach the second line in this test? If you expect the same line in the source code to raise this `AssertionError`, then test coverage wouldn't be enough.

Reply:
I have split the test into two. please refer latest test file.